virtual_rainforest/models/animals/animal_communities.py

```diff
@@ -97,11 +97,16 @@ def die_cohort(self, cohort: AnimalCohort) -> None:
 
         if cohort.is_alive:
             cohort.is_alive = False
-            LOGGER.debug("An animal cohort has died")
+            # LOGGER.debug("An animal cohort has died")
```

Comment:
Did you mean to leave this commented out?

Reply:
I did. Right now all of the animal cohorts die very quickly and the terminal gets clogged with these death reports. I've left it in as it becomes useful again once I'm parametrizing things.

Reply:
This reminded me to start a new issue on a group wide review of log output across model:  #316

Reply:
@TaranRallings - you can also change the LOGGER level to avoid having to comment out messages. Are you calling `vr_run` from the command line? At the moment, we don't have a switch to do that. If not, you can do `LOGGER.setLevel(logging.INFO)` to mute that kind of thing.

Reply:
I think @davidorme's approach is the best one, but it's also fine to leave as is if that's more convenient. It is only one line, after all.

I also think it's useful to make the log level configurable via an environment variable, so you don't have to keep remembering to pass in an extra command-line argument. Here's how I implemented that for another project: https://github.com/ImperialCollegeLondon/FINESSE/blob/main/finesse/logger.py